documentation/src/main/asciidoc/mapper-pojo.asciidoc

```diff
@@ -34,6 +34,34 @@ TODO The getting started section has a link pointing here and expects the sectio
 include a detailed description of how to define and use bridges.
 ////
 
+Starting with Hibernate Search 6, there are three main interfaces for bridges:
+
+* `ValueBridge` can be used for simple use cases when mapping an object's property.
+The `ValueBridge` is applied at the property level using one of the pre-defined `Field` annotations:
+`GenericField`, `FullTextField`, ...
```

Comment:
So maybe add the `@` here too if we decide to go with it.

Reply:
Fixed